components/sensor/tcs34725.rst

```diff
@@ -49,18 +49,13 @@ required to be set up in your configuration for this sensor to work.
 Configuration variables:
 ------------------------
 
-- **red_channel** (*Optional*): Get the percentage of how strongly the red color channel is activated.
-  All options from :ref:`Sensor <config-sensor>`.
-- **green_channel** (*Optional*): Get the percentage of how strongly the green color channel is activated.
-  All options from :ref:`Sensor <config-sensor>`.
-- **blue_channel** (*Optional*): Get the percentage of how strongly the blue color channel is activated.
-  All options from :ref:`Sensor <config-sensor>`.
-- **clear_channel** (*Optional*): Get the percentage of how strongly the clear (without a color filter)
-  channel is activated. All options from :ref:`Sensor <config-sensor>`.
+- **red_channel** (*Optional*): Get the percentage of red color channel in relation to the ``clear_channel``.
+- **green_channel** (*Optional*): Get the percentage the green color channel in relation to the ``clear_channel``.
+- **blue_channel** (*Optional*): Get the percentage of the blue color channel in relation to the ``clear_channel``.
+- **clear_channel** (*Optional*): Get the percentage of the clear (without a color filter)
+  channel in relation to the maximum value for the choosen ``integration_time``. 
 - **illuminance** (*Optional*): Get the total illuminance of the sensor in lx.
-  All options from :ref:`Sensor <config-sensor>`.
```

Comment:
I wouldn't remove this, now it's unclear what you should specify for this setting.

Reply:
ok